exonum/src/runtime/mod.rs

```diff
@@ -191,6 +191,27 @@ impl From<RuntimeIdentifier> for u32 {
     }
 }
 
+impl std::convert::TryFrom<u32> for RuntimeIdentifier {
```

Comment:
`TryFrom` is a part of the public interface, so having an adequate error type here would be useful. I think it makes sense to rather define the transform `u32 -> Result<RuntimeIdentifier, ()>` as a private function for `RuntimeIdentifier`.